### PR TITLE
add `sload` & `sstore` specs

### PR DESCRIPTION
**Pull request by HAOYUatHZ**
based on https://github.com/appliedzkp/zkevm-specs/pull/94

### CODE REVIEW

File: specs/opcode/54SLOAD_55SSTORE.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+| SSTORE_CLEARS_SCHEDULE | 15000 |
+
+## Constraints
+
+1. opcodeId checks
+   1. opId === OpcodeId(0x54) for `SLOAD`
+   2. opId === OpcodeId(0x55) for `SSTORE`
+2. state transition:
+   - gc
+     - `SLOAD`: +5 (2 stack operations + 1 storage reads + 2 access_list reads/writes)
```
Comment (DreamWuGit) [inline specs/opcode/54SLOAD_55SSTORE.md:21]: May I know where the `2 access_list reads/writes` happen ?

File: specs/opcode/54SLOAD_55SSTORE.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+     - `SSTORE`: +8
+       + 2 stack operations
+       + 2 storage reads/writes
+       + 2 access_list reads/writes
+       + 2 gas_refund reads/writes
```
Comment (DreamWuGit) [inline specs/opcode/54SLOAD_55SSTORE.md:26]: IIUC, `gas_refund` happen when reverting occur, right ?

Comment (HAOYUatHZ) [reply specs/opcode/54SLOAD_55SSTORE.md:26]: if reverted, then there will not be `gas_refund`. 

check `gas_refund` here: 
+ https://eips.ethereum.org/EIPS/eip-2200
+ https://eips.ethereum.org/EIPS/eip-2929
+ https://www.evm.codes/about

File: specs/opcode/54SLOAD_55SSTORE.md (hunk 1 above)
Comment (HAOYUatHZ) [reply specs/opcode/54SLOAD_55SSTORE.md:21]: check the "lookup" section below

File: specs/opcode/54SLOAD_55SSTORE.md (hunk 2 above)
Comment (DreamWuGit) [reply specs/opcode/54SLOAD_55SSTORE.md:26]: I see, Thanks the reference !

**Comment by HAOYUatHZ** [comment]
tests fail. need to fix

**Comment by scroll-dev** [comment]
moved to https://github.com/appliedzkp/zkevm-specs/pull/106

**Pull request closed without merging**